dubbo-config/dubbo-config-dynamic/pom.xml

```diff
@@ -0,0 +1,74 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.dubbo</groupId>
+        <artifactId>dubbo-config</artifactId>
+        <version>2.7.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>dubbo-config-dynamic</artifactId>
+    <packaging>jar</packaging>
+    <name>${project.artifactId}</name>
+    <description>The config api module of dubbo project</description>
+    <properties>
+        <skip_maven_deploy>false</skip_maven_deploy>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-common</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.ctrip.framework.apollo</groupId>
+            <artifactId>apollo-client</artifactId>
+            <version>0.10.0-SNAPSHOT</version>
```

Comment:
next version of apollo-client(1.0.0) will be deployed to maven central repository, so that we could use that version later

Reply:
after https://github.com/ctripcorp/apollo/pull/1315 is merged, we will proceed the process to deploy apollo client to maven central repository.

Reply:
apollo-client 1.0.0-SNAPSHOT is now available in sonatype, see [https://oss.sonatype.org/index.html#nexus-search;gav~com.ctrip.framework.apollo~apollo-client~~~](https://oss.sonatype.org/index.html#nexus-search;gav~com.ctrip.framework.apollo~apollo-client~~~), hopefully we will deploy 1.0.0 by next week.

Reply:
Great! 
By the way, does Apollo provide an equivalent restful API? I don't find any docs describing that.

Reply:
apollo-client is now deployed to maven central repo, [follow this link to see more](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.ctrip.framework.apollo%22%20AND%20a%3A%22apollo-client%22)